test/suites/storage_volume_initial_config.sh

```diff
@@ -0,0 +1,103 @@
+test_storage_volume_initial_config() {
+  pool="storage-initial-values"
+  profile="profile-initial-values"
+  image="testimage"
+
+  ensure_import_testimage
+
+  (
+    set -e
```

Comment:
The subshell (`()`) can also probably go. Then you wouldn't need to `set -e` because `main.sh`'s already doing it.